app/js/ffxivcraftmodel.js

```diff
@@ -1731,7 +1795,17 @@ var LevelTable = {
     57: 142, // 142
     58: 145, // 145
     59: 148, // 148
-    60: 150  // 150
+    60: 150, // 150
+    61: 260,
```

Comment:
I think you'll need to add a block for crafterLevel >= 260 in the calculateBaseProgressIncrease function. You can start with the contents from the crafterLevel >= 120 block and adjust as necessary.